src/app/pages/session-detail/session-detail.page.ts

```diff
@@ -162,15 +163,19 @@ export class SessionDetailPage implements OnInit {
       cancel: "Don't delete",
       submit: 'Yes Delete'
     }
-    this.utilService.alertPopup(msg).then(data => {
+    this.utilService.alertPopup(msg).then(async data => {
       if (data) {
-        this.toast.showToast("Will be implemented soon!!", "success")
+        let result = await this.sessionService.deleteSession(this.id);
+        if(result.responseCode == "OK"){
+          this.toast.showToast("Session deleted succesfully","success");
```

Comment:
Are we not getting this message from backend? If not use translate rather than hardcoding